external/EffectiveSource/CMakeLists.txt

```diff
@@ -0,0 +1,11 @@
+set(LIBRARY EffectiveSource)
+
+add_library(${LIBRARY} kerr-circular.c)
+
+target_link_libraries(
+  ${LIBRARY}
+  PRIVATE
+  GSL::gsl
+  )
```

Comment:
Won't matter until we get other dependencies worked out so tracking doesn't tell us we are broken. We could add the header files to track those as well. Up to you :)

Reply:
ok sure added the `add_interface_lib_headers`